syntax = "proto3";

service setChar {
	rpc setChar(setcharmsg) returns (setcharreply) {}
}

service getChar {
	rpc getChar(setcharmsg) returns (getcharreply) {}
}

message setcharmsg {
	string cmd = 1;
	string args = 2;
	string channel = 3;
	string user = 4;
}

message setcharreply {
	string response = 1;
}

message getcharmsg {
	string cmd = 1;
	string args = 2;
	string channel = 3;
	string user = 4;
}

message getcharreply {
	string response = 1;
}
